packages/components/src/utils/property-checkers/check-url.ts

```diff
@@ -2,7 +2,7 @@ export function checkUrl(value: unknown, error: string) {
   if (typeof value !== 'string' && !(value instanceof URL)) throw new Error(error);
 
   try {
-    new URL(value);
+    URL.canParse(value, window.location.origin);
```

Comment:
This always returns `true`. In your component it would be better to use `checkType` instead of `checkUrl`.